### DISCUSSION TITLE
Font pairs

### DISCUSSION THREAD

**Discussion by ivan-kleshnin** [rsms/inter#767] [Q&A]
Hi guys. Thank you for this awesome font and for all the hard work it took to made it!

Maybe it's not the best place to ask but I'd like other people to share their preferred font pairings with Inter. I'm particularly interested in Serif and Mono couplings. 

I totally realize there are endless combinations... but still if you discovered some exceptional pairings – please share :)

P.S. feel free to close this issue if it doesn't fit your guidelines. Or rename the title, etc.

**Comment by rsms** [comment]
Roboto Mono, Inconsolata and Hack all works pretty well with Inter. Roboto Mono in particular has a very similar x-height and cap height.

**Comment by g-cqd** [comment]
I recently tried Inter/IBM Plex Mono pair and in my opinion, it’s the best font pairing that involves monospaced one.

**Comment by highoncarbs** [comment]
EB garamond works well .

**Comment by harrisonlo** [comment]
Is there any recommendations for serif font pairing?

**Comment by g-cqd** [comment]
> I recently tried Inter/IBM Plex Mono pair and in my opinion, it’s the best font pairing that involves monospaced one.

I can update this now, JetBrains Mono is really a good choice in pairing with Inter. You can mind that.

**Comment by g-cqd** [comment]
> Is there any recommendations for serif font pairing?

Something like Georgia or New York (by Apple)... Did not find an other yet.

**Comment by Porges** [comment]
I don’t know if this is really what anyone wants but I’m mixing Inter with Fira Sans Italic since I wanted to use an italic with a little more character. If you increase Fira Sans by ×1.033em it just about matches perfectly. The weight isn’t an exact match but I haven’t fiddled with that yet (Fira Sans doesn’t come as a variable font so you’d need to tweak Inter to match it rather than vice versa).

![image](https://user-images.githubusercontent.com/12575/122317768-2b9f1780-cf72-11eb-9e9d-f175085a5e39.png)

**Comment by Tylannn** [comment]
There is a [blog](https://www.fontpairings.com/inter-font-pairing/) exploered this well, Best serif pairing of Inter is [IBM Plex Serif](https://www.ibm.com/plex/). I agree with their choice and methods, but I don't like IBM Plex Serif much cause it seems not modern enough. 

I'll just recommend New York (By Apple) personally, and I'm curious about Rasmus's recommendation by the way.
